### PR TITLE
Update Background Color to Yellow

### PR DESCRIPTION
**Pull request by jennyxing**
ticket #2347

- tests passed
- VQA finished

### CODE REVIEW

**Review by tlau425 — COMMENTED** [review]
this color is too bright

File: styles.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 body {
     font-family: 'Slackey', cursive;
-    background-color: blue;
+    background-color: hsla(60, 100%, 50%, 0.434);
```
Comment (mainanchung) [inline styles.css:5]: let's approve this!

**Review by Tyalor — COMMENTED** [review]
Hello

File: styles.css (hunk 1 above)
Comment (davonbl-bl) [inline styles.css:5]: This color aligns with the content of the show

Comment (jennyxing) [inline styles.css:5]: the designer asked for this specific color. should I put it into a variable in case we want to use it later on other pages?

**Review by jennyxing — COMMENTED** [review]
summary of the entire review I did
(looks good overall, just a few minor recommendations regarding code styles)

**Review by mainanchung — COMMENTED** [review]
Go for it!!!!

**Comment by meandorca** [comment]
a beauty

**Review by tlau425 — CHANGES_REQUESTED** [review]
requested changes

**Review by meandorca — COMMENTED** [review]
reviewing a change

File: styles.css (hunk 1 above)
Comment (jennyxing) [inline styles.css:5]:
```suggestion
    background-color: #283943;
```

please use hex instead of hsla

**Comment by jennyxing** [comment]
> this color is too bright

I agree, but the designer wanted this color.

**Pull request merged**